src/util/draftutil.ts

```diff
@@ -82,15 +83,20 @@ export const flattenSteps = (steps: Step[], pack: number): FlattenedStep[] => {
   return res;
 };
 
-export const defaultStepsForLength = (length: number): Step[] =>
-  new Array(length)
+export const defaultStepsForLength = (length: number): Step[] => {
```

Comment:
Not a functional change. Using normalizeDraftSteps so we don't have duplicate logic of removing any last "pass" steps